composer.json

```diff
@@ -18,10 +18,10 @@
         }
     ],
     "require": {
-        "typo3/flow": ">=2.0"
+        "neos/flow": "~7.0"
     },
     "autoload": {
-        "psr-0": {
+        "psr-4": {
```

Comment:
We received an error form the Composer repository update:

> Skipped branch flow-7-compatibility, Invalid package information: 
autoload.psr-4 : invalid value (Networkteam\Tika), namespaces must end with a namespace separator, should be Networkteam\Tika\